PluginOracleNet/API/Discover/GetRefreshSchemaForTable.cs

```diff
@@ -9,87 +9,100 @@ namespace PluginOracleNet.API.Discover
     public static partial class Discover
     {
         private const string GetTableAndColumnsQuery = @"
-            SELECT 
-                t.OWNER,
-                t.TABLE_NAME,
-                c.COLUMN_NAME,
-                c.DATA_TYPE,
-                c.DATA_LENGTH,
-                c.DATA_PRECISION,
-                c.DATA_SCALE,
-                c.NULLABLE,
-                CASE
-                    WHEN tc.CONSTRAINT_TYPE = 'P'
-                        THEN 'P'
-                    ELSE NULL
-                END AS CONSTRAINT_TYPE
-            FROM ALL_TABLES t
-                INNER JOIN ALL_TAB_COLUMNS c ON c.OWNER = t.OWNER AND c.TABLE_NAME = t.TABLE_NAME
-                LEFT OUTER JOIN all_cons_columns ccu ON ccu.COLUMN_NAME = c.COLUMN_NAME AND ccu.TABLE_NAME = t.TABLE_NAME AND ccu.OWNER = t.OWNER
-                LEFT OUTER JOIN SYS.ALL_CONSTRAINTS tc ON tc.CONSTRAINT_NAME = ccu.CONSTRAINT_NAME AND tc.OWNER = ccu.OWNER
-                WHERE TABLESPACE_NAME NOT IN ('SYSTEM', 'SYSAUX', 'TEMP', 'DBFS_DATA')
-                AND t.OWNER='{0}' AND t.TABLE_NAME='{1}'
-                ORDER BY t.TABLE_NAME, c.COLUMN_ID";
+SELECT 
+    t.OWNER,
+    t.TABLE_NAME,
+    c.COLUMN_NAME,
+    c.DATA_TYPE,
+    c.DATA_LENGTH,
+    c.DATA_PRECISION,
+    c.DATA_SCALE,
+    c.NULLABLE,
+    CASE
+        WHEN tc.CONSTRAINT_TYPE = 'P'
+            THEN 'P'
+        ELSE NULL
+    END AS CONSTRAINT_TYPE
+FROM ALL_TABLES t
+    INNER JOIN ALL_TAB_COLUMNS c ON c.OWNER = t.OWNER AND c.TABLE_NAME = t.TABLE_NAME
+    LEFT OUTER JOIN all_cons_columns ccu ON ccu.COLUMN_NAME = c.COLUMN_NAME AND ccu.TABLE_NAME = t.TABLE_NAME AND ccu.OWNER = t.OWNER
+    LEFT OUTER JOIN SYS.ALL_CONSTRAINTS tc ON tc.CONSTRAINT_NAME = ccu.CONSTRAINT_NAME AND tc.OWNER = ccu.OWNER
+    WHERE TABLESPACE_NAME NOT IN ('SYSTEM', 'SYSAUX', 'TEMP', 'DBFS_DATA')
+    AND t.OWNER='{0}' AND t.TABLE_NAME='{1}'
+    AND (tc.CONSTRAINT_TYPE IS NULL OR tc.CONSTRAINT_TYPE = 'P')
```

Comment:
This line was added and will have runtime complexity impacts without actually changing anything due to the case statement above.